### DISCUSSION TITLE
field set to singularLabel

### DISCUSSION THREAD

**Discussion by slakk** [laravel/nova-issues#3202] [Q&A] [answered]
i found just use static text on singularLabel method. I need use product name on label page,
how i can do it?
https://prnt.sc/xpms37

**Comment by crynobone** [comment, marked as answer]
I would assume you need to update `resources/lang/vendor/nova/en.json` file and replace `":resource Details: :title"` value to following:

```json
{
  ":resource Details: :title": ":title",
}
```

**Reply by slakk** [reply]
your decision static text again. i try use field name model
